tests/Product_type_unit_test.py

```diff
@@ -0,0 +1,72 @@
+from selenium import webdriver
+import unittest
+import re
+import sys
+
+
+class ProductTest(unittest.TestCase):
+    def setUp(self):
+        # change path of chromedriver according to which directory you have chromedriver.
+        self.driver = webdriver.Chrome('/home/dr3dd/gsoc/chromedriver')
+        self.driver.implicitly_wait(30)
+        self.base_url = "http://localhost:8000/"
+        self.verificationErrors = []
+        self.accept_next_alert = True
+
+    def login_page(self):
+        driver = self.driver
+        driver.get(self.base_url + "login")
+        driver.find_element_by_id("id_username").clear()
+        driver.find_element_by_id("id_username").send_keys('admin')
+        driver.find_element_by_id("id_password").clear()
+        driver.find_element_by_id("id_password").send_keys('admin')
```

Comment:
Credentials have changed to play nicely with Travis.  See the new unit test example for reference.